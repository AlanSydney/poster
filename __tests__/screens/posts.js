import 'react-native';
import renderer from 'react-test-renderer';


describe('PostScreen Test', () => {
  it('Rendered successfully!', () => {
    const tree = renderer.create('<PostScreen />').toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('VirtualizedList Rendered successfully!', () => {
    const tree = renderer.create('<VirtualizedList />').toJSON();
    expect(tree).toMatchSnapshot();
  });
});
