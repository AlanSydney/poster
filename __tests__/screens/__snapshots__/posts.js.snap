// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostScreen Test Rendered successfully! 1`] = `"<PostScreen />"`;

exports[`PostScreen Test VirtualizedList Rendered successfully! 1`] = `"<VirtualizedList />"`;
